lms/djangoapps/grades/tests/utils.py

```diff
@@ -33,18 +34,18 @@ def mock_get_score(earned=0, possible=1):
             weighted_possible=possible,
             weight=1,
             graded=True,
-            attempted=True,
+            first_attempted=datetime(2000, 1, 1, 0, 0, 0)
         )
         yield mock_score
 
 
 @contextmanager
-def mock_get_submissions_score(earned=0, possible=1, attempted=True):
+def mock_get_submissions_score(earned=0, possible=1, first_attempted=datetime(2000, 1, 1, 0, 0, 0)):
     """
     Mocks the _get_submissions_score function to return the specified values
     """
     with patch('lms.djangoapps.grades.scores._get_score_from_submissions') as mock_score:
-        mock_score.return_value = (earned, possible, earned, possible, attempted)
+        mock_score.return_value = (earned, possible, earned, possible, True, first_attempted)
```

Comment:
Sorry.  I thought I had gotten that one.  Running grades tests locally, then I'll push it up.